CONTRIBUTING.md

```diff
@@ -1,3 +1,3 @@
 # Signing the CLA
 
-If you are going to be contributing back to InfluxDB please take a second to sign our CLA, which can be found on our [website](https://influxdata.com/community/cla/).
+If you are going to contribute to InfluxDB, please take a second to sign our [CLA](https://influxdata.com/community/cla/).
```

Comment:
Does it make sense add any of these sections from CONTRIBUTING in influxdb repo?:
- Bug reports
- Feature requests
- Submitting a pull request
- Security vulnerability reporting
- Building, Testing, and Installing

Reply:
Perhaps bug reports / feature requests / and pull requests.

I don't think the others are relevant

Reply:
Added new sections to the CONTRIBUTING file